packages/modular-scripts/src/check/verifyPackageTree.ts

```diff
@@ -47,17 +48,24 @@ export async function check(): Promise<boolean> {
   });
   // Verify we don't have other versions up the tree
   const startDirectory = path.dirname(
-    require.resolve('modular-scripts/package.json'),
+    require.resolve('modular-scripts/package.json', {
+      paths: [getModularRoot()],
+    }),
   );
-
+  let logOutCount = 0;
   for (const dep of DEPS_TO_CHECK) {
     let currentDir = startDirectory;
 
     // We've reached the root of the modular repo
     // in which case we want to bail out
-    while (currentDir !== path.resolve('/')) {
+    // Stop if we've reached the drive's root
```

Comment:
With this change I was able to stop the infinite loop and get the tests to pass - I'm assuming because of the symlinked node modules it was getting to root as it never reached modular root, as 
`equire.resolve('modular-scripts/package.json', {
      paths: [getModularRoot()],
    }),`
would resolve to modular/node_modules, not the temp  modular repo's node_modules 

@cristiano-belloni let me know if this change seems sound to you of if you have any concerns

Reply:
I love that resolving `..` of root yields root itself instead of throwing, that's some kind of unmovable motor paradox that launches our checks into infinity.

I don't see any downside in the extra check since nothing can be parent of root (but itself) and I hope users don't host their Modular projects on `/`; it might even harden us against Windows junction oddities, so good for me.